index.html.in

```diff
@@ -402,28 +402,28 @@
                     </div>
                     <div class="modal-body">
                         <div data-text="What_Is_Apertium"></div>
-                        <div data-text="Maintainer" style="padding-bottom: 2em;"></div>
-
+                        <div data-text="Maintainer" id="aboutModalMaintainer"></div>
+                        
```

Comment:
Trailing whitespace?

Reply:
something went wrong in syncing in vscode sorry.I will re-push it.